doc/bootstrap.rst

```diff
@@ -144,6 +144,27 @@ deploy one or more new manager daemons,::
 Deploying MDSs
 ==============
 
-In order to use the CephFS file system, one or more MDS daemons is needed.
+In order to use the CephFS file system, one or more MDS daemons is
+needed.  Normally these are created automatically if the newer ``ceph
+fs volume`` interface is used to create a new file system.  For more information, see :ref:`fs-volumes-and-subvolumes`.
 
-TBD
+To explicitly deploy metadata servers,::
+
+  # ceph orchestrator mds update *<fs-name>* *<num-daemons>* [*<host1>* ...]
+
+Deploying RGWs
+==============
+
+Cephadm deploys radosgw as a collection of daemons that all manage a
+particular *realm* and *zone*.  (For more information about realms and
+zones, see :ref:`multisite`.)  To deploy a set of radosgw daemons for
+a particular realm and zone,::
+
+  # ceph orchestrator rgw update *<realm-name>* *<zone-name>* *<num-daemons>* [*<host1>* ...]
```

Comment:
There's also the `rgw create` command ... I'm not sure at this point which is the one that is needed to be called.